includes/WikiPage.php

```diff
@@ -2204,7 +2204,7 @@ public function doDeleteArticleReal(
 		$hookAddedLogEntry = false;
 
 		# @todo mediawiki merge 1.19: $log doesn't exist there
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this "TODO" comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)